### PR TITLE
Add 7E chameleon pauldron

### PR DESCRIPTION
**Pull request by novaepee**
## About The Pull Request

Fancy Star Wars shoulder pads that you can wear on either left, right, or both!

## Why It's Good For The Game

image

## Changelog

:cl: novaepee, Fitz A. Sholl
add: 7E chameleon pauldron
/:cl:

### CODE REVIEW

File: code/modules/clothing/modular_armor/attachments/pauldron.dm [hunk 1]
```diff
@@ -0,0 +1,69 @@
+/obj/item/armor_module/greyscale/pauldron
+	name = "7E Chameleon Pauldron"
+	desc = "The 7E Chameloen Pauldron uses brand new and revolutionary technology to make your shoulders look COOLER. It is capable of changing into a variety of different shapes (Alt-Click) and changing colors (Attack with Green Facepaint) to give you that extra edge in style!"
+	greyscale_config = /datum/greyscale_config/pauldron/right
+	icon_state = "in_hand"
+	slot = ATTACHMENT_SLOT_PAULDRON
+	flags_attach_features = ATTACH_REMOVABLE|ATTACH_APPLY_ON_MOB|ATTACH_NO_HANDS|ATTACH_SAME_ICON
+
+	///Current selected pauldron shape. This is the icon_state for the greyscale config used for the mob sprite.
+	var/current_shape = "Standard Right Pauldron"
```
Comment (novaepee) [inline code/modules/clothing/modular_armor/attachments/pauldron.dm:10]: Need review in this.

**Comment by Tailiik** [comment]
Show us what it looks like :hampter:

**Comment by Arctrooper8** [comment]
image
Pictured above: the marine force walking straight into the xeno hive unable to see anything that isn't in front of them.

While I am glad that we're finally giving marines roleplay reasons to engage in their usual intense love for tunnel vision. Don't the jaeger suits already kind of have pauldrons on them? And this feels like a slippery slope towards some goofy World of Warcraft/Warhammer 40k pauldron arms race.

Also no pictures on a sprite PR. We want to see how it actually looks ingame and the icondiff bot isn't really capable of showing that off .

**Comment by novaepee** [comment]
Wait until hyperscale to come back.

**Pull request closed without merging**